### PR TITLE
[CDAP-13697] Improvements for the design of run info header

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: 
https://issues.cask.co/browse/CDAP-13697
https://issues.cask.co/browse/CDAP-13456

Build: https://builds.cask.co/browse/CDAP-URUT30

### CODE REVIEW

File: cdap-ui/app/cdap/components/CollapsibleSidebar/index.js [hunk 1]
```diff
@@ -89,7 +91,7 @@ export default class CollapsibleSidebar extends Component {
             </span>
           </div>
 
-          <div className="collapsible-body">
+          <div className="collapsible-body" onClick={preventPropagation}>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/CollapsibleSidebar/index.js]: nit: separate the props to newline

**Review by elfenheart — COMMENTED** [review]
minor comment, otherwise 👍 code LGTM

**Review by elfenheart — APPROVED** [review]
👍  code LGTM

**Comment by ajainarayanan** [comment]
Demoed to @bdmogal. Merging.

**Pull request merged**